implementations/rust/ockam/ockam_command/src/base_command.rs

```diff
@@ -47,6 +56,7 @@ impl BaseCommand {
         cmd.cluster_init(ctx, &opts).await?;
         let zone_config = cmd.cluster_create(ctx, &opts).await?;
         let inlet_handle = cmd.cluster_inlet(ctx, &opts, &zone_config).await?;
+        // let inlet_handle = cmd.dummy_inlet(&opts).await?;
```

Comment:
commented out code

Reply:
I left it on purpose for now. I just keep adding it/removing it while working in this command.